developer-docs-site/docs/concepts/coin-and-token/aptos-token-overview.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Aptos Token Standard Overview"
+id: "aptos-token-overview"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Aptos Token Standard Overview
+
+:::tip Aptos token standard
+For details, see the official [Aptos Token](./aptos-token.md) standard.
+:::
+
+## Introduction
+One of the major applications of blockchains is token generation and use; this use requires a standard defining
+the pattern a token must comply with. The Aptos team and communities have also been moving
+fast and steadily to support various projects and creators to launch their tokens on the Aptos
+blockchain. As a new L1 blockchain equipped with the [Move programming language](../../guides/move-guides/index.md) - a relatively new language exclusively designed for the safety of smart contracts - our token standard inevitably differs from other
+blockchains more or less. In this document, we will briefly compare the token standard of Aptos with those of Ethereum and Solana.
+
+### Account model
+In order to understand tokens, we have to compare the account model across different
+blockchains. This section contains a brief summary.
+
+#### Ethereum 
+Ethereum has two types of accounts: Externally-owned accounts (EOA) essentially just stores
+a balance of ETH. Contract accounts manage their underlying smart contracts and have an associated 
+storage map that stores all the persisted data, which can be mutated only by the contract, enforced 
+by EVM.
+
+#### Solana
+There are also two types of accounts on the Solana blockchain: executable and non-executable.
+But how the data and code are stored and managed is different. In Ethereum, the data is stored
+under the same contract account that manages this piece of data. In the Solana blockchain,
+the data store and program are separated into different accounts. An executable account
+stores only contract code while all the associated data is stored in non-executable accounts owned
+by the executable account.
+
+#### Aptos
+The [accounts](../accounts.md) in Aptos are homogeneous in that they can store both smart contracts and data. But
```

Comment:
Is homogeneous the right word here?

Reply:
I am not able to come up with a better word that describe there is no differentiation between accounts by their functionalities. Equivalent?

Reply:
Can we say that then?